packages/api/pubspec.yaml

```diff
@@ -11,11 +11,16 @@ dependencies:
   flutter:
     sdk: flutter
   get_it: ^7.2.0
+  objectbox: ^1.6.0
+  objectbox_flutter_libs: any
```

Comment:
Check if you can specify a version number instead of any. Try using the same version as objectbox (^1.6.0)